scripts/generate_runtime.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
add this script into Makefile like `generate_client.sh`

Reply:
> add this script into Makefile like `generate_client.sh`
would solve in subsequent patch

Reply:
> add this script into Makefile like `generate_client.sh`

would solve in subsequent patch

Reply:
I think you should include it in test pipeline in this PR. Check diff or anything else.